### DISCUSSION TITLE
Message from MessageId

### DISCUSSION THREAD

**Discussion by bunchofcellulose** [serenity-rs/serenity#1859] [Q&A] [answered]
Is it possible to get a Message from a MessageId?

**Comment by vicky5124** [comment, marked as answer]
Not exclusively, you will also need the channel_id the message is in.
https://serenity-rs.github.io/serenity/current/serenity/http/client/struct.Http.html#method.get_message

`ctx.http.get_message(channel_id, message_id).await?`
